ledger_sync.gemspec

```diff
@@ -53,7 +53,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency('faraday_middleware', '>= 0')
   spec.add_runtime_dependency('fingerprintable', '>= 1.2.1')
   spec.add_runtime_dependency('nokogiri', '>= 0')
-  spec.add_runtime_dependency('openssl', '~> 2.2.0')
+  spec.add_runtime_dependency('openssl', '~> 3.0')
```

Comment:
I think we should just do `> 2.2.0` for now. This could be a breaking change for some. There's some wonkiness with the openssl version, which is why it's even included here. I haven't found the root cause, but it seems related to the ruby 2 > 3 migration. So I don't want to disallow openssl 2.2 but we can open it up to allow 3+. Does that work?